test_collections/sdk_tests/support/tests/python_tests/test_utils.py

```diff
@@ -49,7 +49,7 @@ async def test_generate_command_arguments_on_network() -> None:
         "--passcode 1234",
         "--commissioning-method on-network",
         "--paa-trust-store-path /paa-root-certs",
-        "--storage-path /root/admin_storage.json",
+        "--storage_path /root/admin_storage.json",
```

Comment:
do we need to replace "-" with "_" here?

Reply:
maybe add a comment about that here @rquidute

Reply:
Done